components/layout_2020/fragment_tree/box_fragment.rs

```diff
@@ -192,6 +192,35 @@ impl BoxFragment {
         )
     }
 
+    pub fn clip_overflow(&self, containing_block_rect: &PhysicalRect<Au>) -> PhysicalRect<Au> {
+        let mut clip_rect = self
+            .padding_rect()
+            .translate(containing_block_rect.origin.to_vector());
+
+        // Adjust by the overflow clip margin.
+        // https://drafts.csswg.org/css-overflow/#overflow-clip-margin
+        let clip_margin = Au::from(self.style.get_margin().overflow_clip_margin);
+        clip_rect.size.width += clip_margin;
+        clip_rect.size.height += clip_margin;
+
+        let mut overflow = self
+            .scrollable_overflow()
```

Comment:
Per spec:

> The scrollable overflow of a box is the set of things extending outside of that box’s padding edge for which a scrolling mechanism needs to be provided.

`overflow: clip` doesn't need scrolling mechanism. I'm wondering is there still any scrollable overflow? And is Servo handling this case correctly?